docs/reference/api-conventions.asciidoc

```diff
@@ -42,6 +42,19 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=expand-wildcards]
 
 The defaults settings for the above parameters depend on the API being used.
 
+Some indices (hereafter "system indices") are used by various system
+components and/or plugins to store state or configuration. These indices
```

Comment:
Do we define what a "system component" is?

Reply:
This should probably be `modules and/or plugins`, thanks.